README.md

```diff
@@ -1 +1,583 @@
+- Identifying the unavailble services
+- Determining database call duration and bottleneck queries
+- Function execution time (user-defined observability)
+
+Consider a real-world use case of a travel agency that arranges complete tours for users. A tour package includes airline ticket reservation, hotel room reservation and car rental. Therefore, the travel agency service requires communicating with other necessary back-ends. Those back-ends query their databases to retrieve information of availability of their services.
+
+Travel Agency service communicates with multiple services. Airline reservation service has multiple resources for three different airlines, whereas Hotel reservation and Car rental services only have a single resource. The travel agency service checks all airline resources in parallel to select the cheapest available. Each associated service then queries their database for the availability and information of their services. Only Hotel resevation service does not have a associated database. Rather the response is hard-coded to demonstrate the difference in traces.
+
+![Ballerina Observability Sample](/images/ballerina-observability-sample.png)
+
+## Prerequisites
+ 
+- [Ballerina Distribution](https://ballerina.io/learn/getting-started/)
+- A Text Editor or an IDE
+- [Docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+- [Prometheus](https://ballerina.io/learn/how-to-observe-ballerina-code/#prometheus)
+- [Grafana](https://ballerina.io/learn/how-to-observe-ballerina-code/#grafana)
+- [Jaeger](https://ballerina.io/learn/how-to-observe-ballerina-code/#jaeger-server)
+
+### Optional requirements
+- Ballerina IDE plugins ([IntelliJ IDEA](https://plugins.jetbrains.com/plugin/9520-ballerina), [VSCode](https://marketplace.visualstudio.com/items?itemName=WSO2.Ballerina), [Atom](https://atom.io/packages/language-ballerina))
+- [Kubernetes](https://kubernetes.io/docs/setup/)
+
+## Testing the services
+
+### Setting up the environment  
+
+Please refer to the [Prerequisites](#prerequisites) section for information on setting up the required tools and environment. Access the below URLs and make sure you have Prometheus, Grafana and Jaeger servers up and running.
+- Prometheus - [http://localhost:19090](http://localhost:19090)
+- Grafana - [http://localhost:3000](http://localhost:3000)
+- Jaeger - [http://localhost:16686](http://localhost:16686)
+
+Create the database using the script provided in resources directory. This datbase is being used by the services to retrieve data.
+```sql
+mysql -u <username> -p
+source resources/mysql.sql;
+```
+
+### Invoking the service
+
+- Navigate to `microservices-observability/guide` and run the following commands in separate terminals to start all four HTTP services. This will start the `Airline Reservation`, `Hotel Reservation`, `Car Rental` and `Travel Agency` services in ports 9091, 9092, 9093 and 9090 respectively.
+
+```bash
+   $ ballerina run travel_agency/
```

Comment:
Shall we get rid of the trailing `/`? It's not part of the package name. Also, I'm not sure of the use of `_` in the package name.

Reply:
Fix the above in other relevant places as well